activerecord/test/cases/migration/columns_test.rb

```diff
@@ -321,6 +321,26 @@ def test_add_column_without_column_name
       ensure
         connection.drop_table :my_table, if_exists: true
       end
+
+      def test_remove_columns_single_statement
+        connection.create_table "my_table" do |t|
+          t.integer "col_one"
+          t.integer "col_two"
+        end
+
+        queries = capture_sql { connection.remove_columns("my_table", "col_one", "col_two") }
+
+        columns = connection.columns("my_table").map(&:name)
+        assert_equal ["id"], columns
+
+        if current_adapter?(:Mysql2Adapter, :PostgreSQLAdapter)
```

Comment:
Maybe check `supports_bulk_alter?` instead?

Reply:
Before submitting the PR, I flip flopped on this choice a bit.

On one hand, using `supports_bulk_alter?` kind of copies and reuses the code under test, so using an explicit list of adapters here shows exactly which ones this applies to and there is less confusion on the side effects of the change.

On the other hand, as adapters improve, this test could fail requiring some extra maintenance. 

Ultimately, I'm happy to go either way, just explaining my thought process.

Reply:
Fixed in "Use supports_bulk_alter? in tests"